crates/ika-core/src/dwallet_mpc/dwallet_mpc_service.rs

```diff
@@ -59,6 +52,7 @@ impl DWalletMPCService {
         let dwallet_mpc_manager = DWalletMPCManager::must_create_dwallet_mpc_manager(
             consensus_adapter.clone(),
             epoch_store.clone(),
+            network_keys_receiver,
```

Comment:
mpc manager now receives network keys, which makes sense because it handles event and some event require the network key